### DISCUSSION TITLE
Migration guide from 2.8.3 to 3

### DISCUSSION THREAD

**Discussion by Maesla** [MixedRealityToolkit/MixedRealityToolkit-Unity#556] [Q&A]
Hello,

Is there documentation or step by step guide that can be recommended to upgrade a project using MRTK 2.8.3 to MRTK 3?

Thank you

**Comment by anonymous2585** [comment]
Hi,

You have this page: https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-overview/architecture/mrtk-v2-to-v3
But I feel it lack some more advanced informations...

There also are great articles on the blog of @LocalJoost : https://localjoost.github.io/ (exemple: https://localjoost.github.io/Upgrading-to-MRTK3-GA-from-a-pre-release-some-assembly-required/)

Cheers

**Reply by Maesla** [reply]
Thanks!

Yes, I knew the official one from Microsoft but I think it is not very user friendly.

From localjoost, I knew also this one:
https://localjoost.github.io/MRTK2-to-MRTK3-migrating-the-QRCode-sample/

If anyone else knows more resources, please share :)
